docs/reference/ingest/search-ingest-pipelines.asciidoc

```diff
@@ -0,0 +1,313 @@
+
+[WARNING]
+=========================
+You should not rename this pipeline.
+=========================
+
+[discrete#ingest-pipeline-search-upgrading-notes]
+=== Upgrading notes
+
+.Expand to see upgrading notes
+[%collapsible%closed]
+=============
+
+* `app_search_crawler` - Since 8.3, {app-search-crawler} has utilized this pipeline to power its binary content
+extraction.
+  You can read more about this pipeline and its usage in the {app-search-ref}/web-crawler-reference.html#web-crawler-reference-binary-content-extraction[App Search Guide].
+  When upgrading from 8.3 to 8.5+, be sure to note any changes that you made to the `app_search_crawler` pipeline.
+  These changes should be re-applied to each index's `<index-name>@custom` pipeline in order to ensure a consistent data processing experience.
+  In 8.5+, the <<ingest-pipeline-search-pipeline-settings, index setting to enable binary content>> is required *in addition* to the configurations mentioned in the {app-search-ref}/web-crawler-reference.html#web-crawler-reference-binary-content-extraction[App Search Guide].
+
+* `ent_search_crawler` - Since 8.4, the Elastic web crawler has utilized this pipeline to power its binary content extraction.
+  You can read more about this pipeline and its usage in the {enterprise-search-ref}/crawler-managing.html#crawler-managing-binary-content[Elastic web crawler Guide].
+  When upgrading from 8.4 to 8.5+, be sure to note any changes that you made to the `ent_search_crawler` pipeline.
+These changes should be re-applied to each index's `<index-name>@custom` pipeline in order to ensure a consistent data processing experience.
+  In 8.5+, the <<ingest-pipeline-search-pipeline-settings, index setting to enable binary content>> is required *in addition* to the configurations mentioned in the {enterprise-search-ref}/crawler-managing.html#crawler-managing-binary-content[Elastic web crawler Guide].
+
+* `ent-search-generic-ingestion` - Since 8.5, Native Connectors, Connector Clients, and new (>8.4) Elastic web crawler indices will all make use of this pipeline by default.
+  A natural evolution from `app_search_crawler` and `ent_search_crawler`, this pipeline is designed to meet more generic needs across *Search*.
+  You can <<ingest-pipeline-search-details-generic-reference, read more about this pipeline>> above.
+  As this pipeline is "managed", any modifications that were made to `app_search_crawler` and/or `ent_search_crawler` should NOT be made to `ent-search-generic-ingestion`.
+  Instead, if such customizations are desired, you should utilize <<ingest-pipeline-search-details-specific>>, placing all modifications in the `<index-name>@custom` pipeline(s).
+=============
+
+[discrete#ingest-pipeline-search-faq]
+== FAQ
+
+.Expand to see FAQ
+[%collapsible%closed]
+=============
+
+*Q*: The Elasticsearch docs note that there are three types of ingest pipelines.
```

Comment:
I suggest revising this, since we're in the Elasticsearch docs now.

Reply:
Yeah I collapsed the upgrade and faq sections but originally I was wondering if we could remove these entirely at this stage given we're in 8.11+

Reply:
@chriscressman I would vote that we remove the FAQ here I don't think it's very useful at this stage

Reply:
:+1: Sounds good to me. I agree the value is low.